src/components/ProfileForm/components/ProfileFormStep3.tsx

```diff
@@ -0,0 +1,162 @@
+              value="summer_fridays"
+              type="checkbox"
+            />
+            <span className="pl-1"> Summer Fridays </span>
+          </label>
+          <label>
+            <input name="skincare_brands" value="other" type="checkbox" />
+            <span className="pl-1"> Other </span>
+            <input name="skincare_brands" type="text" className="border" />
+          </label>
+        </fieldset>
+        <fieldset className="mb-2 flex flex-col">
+          <legend className="mb-2 font-bold"> Makeup brands you use </legend>
+          <label>
+            <input name="makeup_brands" value="dior" type="checkbox" />
+            <span className="pl-1"> Dior </span>
+          </label>
+          <label>
+            <input name="makeup_brands" value="chanel" type="checkbox" />
+            <span className="pl-1"> Chanel </span>
+          </label>
+          <label>
+            <input name="makeup_brands" value="fenty_beauty" type="checkbox" />
+            <span className="pl-1"> Fenty Beauty </span>
+          </label>
+          <label>
+            <input name="makeup_brands" value="estee_lauder" type="checkbox" />
+            <span className="pl-1"> Estee Lauder </span>
+          </label>
+          <label>
+            <input name="makeup_brands" value="rare_beauty" type="checkbox" />
+            <span className="pl-1"> Rare Beauty </span>
+          </label>
+          <label>
+            <input name="makeup_brands" value="nars" type="checkbox" />
+            <span className="pl-1"> Nars </span>
+          </label>
+          <label>
+            <input name="makeup_brands" value="laura_mercier" type="checkbox" />
+            <span className="pl-1"> Laura Mercier </span>
+          </label>
+          <label>
+            <input name="makeup_brands" value="benefit" type="checkbox" />
+            <span className="pl-1"> Benefit </span>
+          </label>
+          <label>
+            <input name="makeup_brands" value="milk_makeup" type="checkbox" />
+            <span className="pl-1"> Milk Makeup </span>
+          </label>
+          <label>
+            <input name="makeup_brands" value="Tower 28" type="checkbox" />
+            <span className="pl-1"> Tower 28 </span>
+          </label>
+          <label>
+            <input
+              name="makeup_brands"
+              value="makeup_forever"
+              type="checkbox"
+            />
+            <span className="pl-1"> Makeup Forever </span>
+          </label>
+          <label>
+            <input
+              name="makeup_brands"
+              value="charlotte_tilbury"
+              type="checkbox"
+            />
+            <span className="pl-1"> Charlotte Tilbury </span>
+          </label>
+          <label>
+            <input name="makeup_brands" value="other" type="checkbox" />
+            <span className="pl-1"> Other </span>
+            <input name="makeup_brands" type="text" className="border" />
```

Comment:
This is weird. We cannot have 2 inputs with the same name, unless they are `radio` buttons and we want to make sure to only click one.